src/latex.jl

```diff
@@ -9,7 +9,7 @@ latex(text::LaTeXString, x, y) = latex(text, Point(x, y), :stroke)
 latex(text::LaTeXString, x, y, valign::Symbol, halign::Symbol) =
     latex(text, Point(x, y), :stroke, valign = valign, halign = halign)
 """
-    latex(text::LaTeXString, pos::Point, object::Symbol)
+    latex(text::LaTeXString, pos::Point, object::Symbol, valign::Symbol, halign::Symbol)
```

Comment:
they should be keyword arguments here as well.

Reply:
not fixed yet.

Reply:
Oh sorry, I thought I had made the change. Anyways, I pushed recently again to improve codecov and they are reflected in them. Thanks.

Reply:
seems to be still wrong on GitHub, right?